cdc/scheduler/internal/v3/replication/replication_manager.go

```diff
@@ -487,6 +499,8 @@ func (r *Manager) AdvanceCheckpoint(
 				zap.Int64("tableID", tableID))
 			return checkpointCannotProceed, checkpointCannotProceed
 		}
+		// find the slow tables
+		heap.Push(&r.slowTableHeap, table)
```

Comment:
Should the heap be reset at the beginning of the loop?

Reply:
we can only call `heap.Init(&h)` when it is initialized.